front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/index.ts

```diff
@@ -87,5 +87,5 @@ async function handler(
 }
 
 export default withPublicAPIAuthentication(
-  withResourceFetchingFromRoute(handler, "space")
+  withResourceFetchingFromRoute(handler, { space: { requireCanList: true } })
```

Comment:
All these spaces endpoints on v1 should be requireCanRead instead of requireCanList no?

Reply:
The previous space resolver was checking canList only, so I kept it as is (but now it's explicit and must be specified in the options). For this one I think it should still be canList ?

Reply:
Yes this one is canList but most others are canRead IMHO. Will let you review and assign and I'll review once modified :+1:

Reply:
ok I reviewed the endpoints and took the same permission check than the one which are done in the handlers directly. We have some canRead but also a few canWrite